### DISCUSSION TITLE
Compatibility between Dax Studio v. 2.17.3 and SSAS 2022

### DISCUSSION THREAD

**Discussion by matteolocci** [DaxStudio/DaxStudio#1183] [Q&A]
Hi all,
Does anyone know if Dax Studio v. 2.17.3 is compatible with SSAS 2022 and, as a consequence, with level 1600?
Thanks a lot.

Best Regards,
Matteo

**Comment by dgosbell** [comment]
It's pretty hard to say. v2.17.3 was released a year before SSAS 2022. If it does not work the simplest solution would be to just use the latest release of DAX Studio.
